shared-data/protocol/types/schemaV8/index.ts

```diff
@@ -120,7 +132,7 @@ export type ProtocolFile<
 > = ProtocolBase<DesignerApplicationData> &
   (OT2RobotMixin | OT3RobotMixin) &
   LabwareV2Mixin &
-  LiquidV1Mixin &
+  LiquidV2Mixin &
```

Comment:
i _think_ since liquidV1 should still be supported in schema 8, we should support either type

```suggestion
  (LiquidV1Mixin | LiquidV2Mixin) &
```

Reply:
speaking of which, the command types should go all the way up to V14 now right? 😅 thats a separate issue though that we can address in a follow up

Reply:
Agreed that this should be a union of liquid schema 1 and 2